### ISSUE TITLE
point www.innovateeugene.com somewhere

### ISSUE THREAD

**Issue by markdav-is** [EugTech/innovate-eugene-web#6]
@mckelveygreg would you like the www pointing to the API server or something else?

**Comment by mckelveygreg** [comment]
have it point to @johnrnelson 's server. It is setup to host the gatsby build and reroute /api calls.
Currently, it is as http://demo.johnrnelson.com

**Comment by johnrnelson** [comment]
We now have the DNS working so the new server is https://api.innovateeugene.com thanks to @nohorse.

Notice that the **HTTPS** is now in play so be sure to factor that in when debugging. :-) 

I'm not sure about the http://www.innovateeugene.com/ domain but the *api* is working.

**Comment by markdav-is** [comment]
it's confusing to have the sample app and the API on the same web. they should be isolated. if we can static host the www somewhere and call the API for data that would be best. I"m OK with github pages for now and using some public testing credentials.  eventually we'll  want to host it somewhere with its own secrets, so netlify is a good goal. if you want a eugenetech netlify just make an issue.

**Comment by johnrnelson** [comment]
I've been using https://www.apitester.com/ to test the api with just to be sure it's open and easy. I removed the HTTP verb completely to help clear up any confusion. 

We just keep the web and API on the same server for costs but as soon as it's hosted somewhere else they can easily break it up.

I also stopped putting changes directly into the github project. I forked it, and will do the pull requests like you suggested we do weeks ago. So far so good. I did my first merge this morning.
